### PR TITLE
fix: disabled edit/create profile form validation on change

### PR DESCRIPTION
**Pull request by wawrzonnn**

### CODE REVIEW

File: src/components/CreateProfileForm/CreateProfileFormWrapper.tsx [hunk 1]
```diff
@@ -101,6 +101,8 @@ const CreateProfileFormWrapper = ({ children }: PropsWithChildren) => {
       initialValues={initialValues}
       validationSchema={validationSchema}
       onSubmit={handleCreateProfile}
+      validateOnBlur={false}
+      validateOnChange={false}
```
Comment (Sumick) [inline src/components/CreateProfileForm/CreateProfileFormWrapper.tsx:105]: **issue:** You disabled live validation completely now - it should validateOnBlur, and validateOnChange but only when form is submitted once - we should use `touched` property to manage that in inputs

**Review by Sumick — CHANGES_REQUESTED** [review]
We should not disable validation completely, it should work after first submission try

**Review by AniaPiWo — APPROVED** [review]

**Review by hansac91 — APPROVED** [review]

**Review by mmcode9407 — APPROVED** [review]

File: app/(profiles)/my-profile/create/page.tsx [hunk 2]
```diff
@@ -21,9 +21,9 @@ const CreateProfilePage = async () => {
 
   const profile = await getProfileByUserEmail(session.user.email)
 
-  if (profile) {
-    redirect(AppRoutes.myProfile)
-  }
+  // if (profile) {
+  //   redirect(AppRoutes.myProfile)
```
Comment (Sumick) [inline app/(profiles)/my-profile/create/page.tsx:25]: **nitpick:** Comment to remove?

File: src/components/CreateProfile/LocationPreferences/LocationPreferences.tsx [hunk 3]
```diff
@@ -37,8 +41,9 @@ const LocationPreferences = () => {
           />
         </div>
         <div>
-          <InputFormError error={errors.city}>
+          <InputFormError error={errors.city} touched={touched.city}>
```
Comment (Sumick) [inline src/components/CreateProfile/LocationPreferences/LocationPreferences.tsx:44]: **nitpick:** We don't need to pass `touched` here. You can just check it in `error field`, so it's:

```tsx
  <InputFormError error={touched.city && errors.city && errors.city}>
```

File: src/components/CreateProfile/LocationPreferences/LocationPreferences.tsx [hunk 4]
```diff
@@ -59,8 +65,9 @@ const LocationPreferences = () => {
             dataTestId="openToRelocationCity"
           />
         </div>
-        <InputFormError error={errors.remoteOnly}>
+        <InputFormError error={errors.remoteOnly} touched={touched.remoteOnly}>
```
Comment (Sumick) [inline src/components/CreateProfile/LocationPreferences/LocationPreferences.tsx:68]: **nitpick:** Same here

File: src/components/CreateProfile/PersonalInfo/PersonalInfo.tsx [hunk 5]
```diff
@@ -28,8 +28,10 @@ const PersonalInfo = ({ profile }: PersonalInfoProps) => {
         </div>
       </div>
       <div className={styles.right}>
-        <InputFormError error={errors.fullName}>
+        <InputFormError error={errors.fullName} touched={touched.fullName}>
+          {touched.fullName && <span>huj</span>}
```
Comment (Sumick) [inline src/components/CreateProfile/PersonalInfo/PersonalInfo.tsx:32]: **issue:** xD To remove

File: src/components/InputFormError/InputFormError.tsx [hunk 6]
```diff
@@ -3,13 +3,14 @@ import styles from './InputFormError.module.scss'
...
 }
 
-const InputFormError = ({ error, children }: InputWrapperProps) => {
+const InputFormError = ({ error, children, touched }: InputWrapperProps) => {
   return (
-    <div className={error ? styles.errorMsg : ''}>
+    <div className={error && touched ? styles.errorMsg : ''}>
       {children}
-      {error && <p>{error}</p>}
+      {error && touched && <p>{error}</p>}
```
Comment (Sumick) [inline src/components/InputFormError/InputFormError.tsx:13]: **nitpick:** We don't need this prop here

**Review by Sumick — CHANGES_REQUESTED** [review]
Mostly looks good, but let's do a little cleanup before merging - especially let's remove all`touched` from inputs level, and leave just error message

**Review by Sumick — APPROVED** [review]
Goood job!

**Pull request merged**